tests/test_icml_conference.py

```diff
@@ -5722,14 +5722,30 @@ def test_forum_chat(self, openreview_client, helpers):
         ## Add tag emoji
         tag = sac_client.post_tag(openreview.api.Tag(
             invitation='ICML.cc/2023/Conference/Submission1/-/Chat_Reaction',
-            signatures=['ICML.cc/2023/Conference/Submission1/Senior_Area_Chairs'],
-            tag='😄',
-            replyto=note_edit['note']['id']
+            signature='ICML.cc/2023/Conference/Submission1/Senior_Area_Chairs',
```

Comment:
does it mean we need to update the current tag invitations?

Reply:
Yes, https://github.com/openreview/openreview-api/pull/627.

I am open to suggestions, of course